### PR TITLE
[DOC] feature files structure

### PR DESCRIPTION
**Pull request by iariasleon**

### CODE REVIEW

File: test/acceptance/behave/README.md [hunk 1]
```diff
@@ -81,6 +81,7 @@ pip install --upgrade -r requirements.txt --allow-all-external
     properties.json.base:                reference file with parameters (properties) used in tests (after is copied to properties.json)
     properties.json:                     initially does not exists. This has parameters necessary to execute the tests (see [properties.json.base](https://github.com/telefonicaid/fiware-orion/tree/develop/test/acceptance/behave#propertiesjsonbase))
     README.md:                           this file, a brief explication about this framework to test
+    feature_files_structure.md           this file, a brief explication about the feature files structure
```
Comment (fgalan) [inline test/acceptance/behave/README.md]: "this file" refers to README.md, so I understand it should be removed.

**Comment by jcanonav** [comment]
LGTM

**Comment by XavierVal** [comment]
LGTM

File: test/acceptance/behave/README.md (hunk 1 above)
Comment (iariasleon) [inline test/acceptance/behave/README.md]: Fixed in 79e9ac1

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**